src/main/java/ru/yandex/practicum/filmorate/advice/ErrorHandler.java

```diff
@@ -79,4 +75,22 @@ public ErrorResponse handleConstraintViolationException(final ConstraintViolatio
                 e.getMessage()
         );
     }
+
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ErrorResponse handleGenreNotFoundException(final ErrorGenreException e) {
```

Comment:
Лучше отказаться от именований переменных в одну букву, так код выглядит менее читаемым
```suggestion
    public ErrorResponse handleGenreNotFoundException(final ErrorGenreException exception) {
```